kahuna/public/js/image/service.js

```diff
@@ -8,6 +10,7 @@ imageService.factory('imageService', [function() {
     function forImage(image) {
```

Comment:
I really don't like this helper that does loads of extra work for nothing. Even if not now, we should change it to individual helpers for each transformation we're interested in.